tania-prof.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE hmtl>
+<hmtl>
+
+    <head>
+        <link rel="stylesheet" type="text/css" href="h1 {.css">
+        <!-- Add icon library -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+        <title>Tania Yeroman's Personal Profile</title>
+    </head>
+    <main>
+
+        <body>
+            <nav>
+                <div class="navbar">
```

Comment:
You could have styled 'nav a' instead of creating a div to wrap your anchor tags in. This would minimise amount of code necessary.